### PR TITLE
[3928] Reactify History dialog(adding Monaco)

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/3928

Adding monaco to public folder

### CODE REVIEW

**Review by rart — CHANGES_REQUESTED** [review]
The idea is to add Monaco directly to static assets like other libs. Not as app assets that need to be built and bundled.

Let's see if we can pull from NPM and copy in a yarn task. That way we needn't check in the source 🙂

File: ui/app/public/index.html [hunk 1]
```diff
@@ -30,8 +30,16 @@
...
   >
+  <link
+    href="/js/monaco/vs/editor/editor.main.css"
+    rel="stylesheet"
+  >
 </head>
 <body>
+<script>var require = { paths: { 'vs': '/js/monaco/vs' } };</script>
+<script src="/js/monaco/vs/loader.js"></script>
+<script src="/js/monaco/vs/editor/editor.main.nls.js"></script>
```
Comment (rart) [inline ui/app/public/index.html:41]: What's this one for?

Comment (aachavarria) [reply ui/app/public/index.html:41]: to load monaco editor, it needs that files

Comment (rart) [reply ui/app/public/index.html:41]: yes, but what is this one specifically for? (main.nsl.js)

Comment (aachavarria) [reply ui/app/public/index.html:41]: I think it contains the languages

**Pull request closed without merging**